backend/routes/users.ts

```diff
@@ -63,6 +63,34 @@ router.patch('/:userId', async (req: any, res: any) => {
   }
 });
 
+router.put('/:userId/test', async (req: any, res: any) => {
+  try {
+    const { userId } = req.params;
+    const updates = req.body;
+    const updatedUser = await User.findByIdAndUpdate(userId, {
+      $inc: { totalHours: updates.numHours },
+    });
+    res.json(updatedUser);
+  } catch (error) {
+    res.status(400).send(error);
+  }
+});
+
+// put new shift into user's past shifts
+router.put('/:userId', async (req: any, res: any) => {
+  try {
```

Comment:
Should one of these be removed?  
Whichever one you keep, make sure it's name does not include the extra `/test` in the path